scripts/travis/linux-build-checks/check-fmt.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+set -euxo pipefail
+
+scons -Q fmt
+
+diff=$(git status --porcelain)
+
+if [[ $diff ]]; then
+    echo $diff
+    echo "format check FAILED"
+    echo "please install clang-format and run scons -Q fmt"
+    exit 1
```

Comment:
I suggest a small improvement:

```
if [[ $diff ]]; then
    echo
    echo "format check FAILED"
    echo "please install clang-format and run scons -Q fmt"
    echo
    GIT_PAGER= git diff
    exit 1
```

The output will be a bit more convenient IMHO.

![image](https://user-images.githubusercontent.com/8436629/78007853-89cce400-7347-11ea-927d-c603198f82ef.png)